src/apiCalls.js

```diff
@@ -0,0 +1,17 @@
+
+export const getPlaces = (url) => {
+    return fetch(url)
+      .then(res => {
+        if (!res.ok) {
+          throw new Error('Oops! We seem to be having some technical issues, please try again later!');
+        }
+        return res.json();
+      })
+      .then(placesData => {
+        return placesData;
```

Comment:
I'm curious about this return on line 11. Is this necessary since there is already the return on line 3 that is returning a promise? I could have that wrong though. Maybe worth checking out.

Reply:
Agreed with Patrick here. The `.then` block lines 10-12 is not needed in this function because the final `.then` action step occurs back in the App component.